src/ontology/oba.Makefile

```diff
@@ -139,14 +160,28 @@ $(REPORTDIR)/%.tsv: ../sparql/synonyms-exact.sparql $(TMPDIR)/%.owl
 prepare_oba_alignment: $(REPORTDIR)/uberon.tsv $(REPORTDIR)/efo.tsv $(REPORTDIR)/pato.tsv $(REPORTDIR)/oba.tsv $(REPORTDIR)/vt.tsv $(REPORTDIR)/cl.tsv $(REPORTDIR)/go.tsv $(REPORTDIR)/chebi.tsv
 	echo "OK cool all tables prepared."
 
-CHECK_SPARQL=oba.owl
+ONTOLOGIES=vt oba
+ONTOLOGIES_DIR=data/ontologies
+OWL_FILES=$(foreach o,$(ONTOLOGIES), $(ONTOLOGIES_DIR)/$(o).owl)
+
+DB_FILES=$(foreach o,$(ONTOLOGIES), $(ONTOLOGIES_DIR)/$(o).db)
+OWL_SOURCES=$(patsubst $(ONTOLOGIES_DIR)/%.owl, http://purl.obolibrary.org/obo/%.owl,$@)
+
+$(OWL_FILES):
+	@[ -d $(ONTOLOGIES_DIR) ] || mkdir -p $(ONTOLOGIES_DIR)
+	@wget $(OWL_SOURCES) -P $(ONTOLOGIES_DIR)
+
+
+$(DB_FILES): $(OWL_FILES)
+	@semsql make $@
+
+
+.PHONY: download db
+
+download: $(OWL_FILES)
+db: $(DB_FILES)
 
-check_children_oba: $(CHECK_SPARQL)
-	$(ROBOT) verify -i $< --queries ../sparql/biological-attribute-child-violation.sparql -O $(REPORTDIR)
 
-test: check_children_oba
+build: download db
 
-oba_reports: $(CHECK_SPARQL)
-	sh run.sh robot query -i $< \
-		--query ../sparql/oba-pato-report.sparql reports/oba-pato.csv \
-		--query ../sparql/oba-grouping-report.sparql reports/oba-grouping.csv
\ No newline at end of file
+.DEFAULT_GOAL = build
```

Comment:
Do not change the default goal of this Makefile :)

Reply:
Got it :D